From 56b421a18ef8acb41ab9e4bde7e1f5452ec67da9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 17 May 2021 22:17:07 -0700
Subject: [PATCH] setup.py: Correct the license assignment

latest python/2.9 + setuptools bail out otherwise

lines = header.split('\n')
| AttributeError: 'list' object has no attribute 'split'

Upstream-Status: Submitted [https://github.com/mapado/haversine/pull/38]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 549436d..9e7fb00 100755
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@ setup(
     maintainer_email='julien.deniau@mapado.com',
     url='https://github.com/mapado/haversine',
     packages=['haversine'],
-    license=['MIT'],
+    license='MIT',
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
-- 
2.31.1

